roles/tas_single_node/templates/configs/ctlog-config-template.j2

```diff
@@ -10,6 +10,53 @@ log_configs:{
                 path: "/ctfe-keys/{{ log.private_key }}"
                 password: "{{ log.password }}"
                 }}
-        ext_key_usages:"CodeSigning"  log_backend_name:"trillian"}
+        ext_key_usages: "CodeSigning"  
+        log_backend_name: "trillian"
+        {% if log.get('not_after_limit') -%}
+        not_after_limit: {{ log.not_after_limit | to_json | regex_replace('"([^"]+)":', '\\1:') | regex_replace('"(\d+)"', '\\1') }}
+        {% endif -%}
+        {% if log.get('public_key') -%}
+        public_key: {der:"{{ log.public_key }}"}
```

Comment:
I would assume most users would provide the key in PEM format; we should either call the DER format out explicitly in the docs (or do the conversion ourselves).

Reply:
Only DER format is supported in this configuration